Watchman.Discord/Areas/Responses/Services/ResponsesMessageService.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Devscord.DiscordFramework.Commons.Extensions;
+using Devscord.DiscordFramework.Framework.Commands.Responses;
+using Devscord.DiscordFramework.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Watchman.Discord.Areas.Responses.Services
+{
+    public class ResponsesMessageService
+    {
+        private const string DESCRIPTION = "dokumentacja:\nhttps://watchman.readthedocs.io/pl/latest/135-services-in-framework/";
```

Comment:
tak na przyszłość -> dobrze byłoby wrzucać takie informacje do bazy, żeby to było łatwo konfigurowalne
bo co jak link sie zmieni?

ale w tym zadaniu może jeszcze tak zostać